### PR TITLE
JNG-4474 release judo spring boot starter

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4474" title="JNG-4474" target="_blank">Sub-taskJNG-4474</a>  Release judo-spring-boot-starter
  </td></table>
  <br />

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -119,22 +119,22 @@ jobs:
...
-          sonar:sonar
+#      - name: 🔎 Sonar Metrics
+#        if: steps.context.outputs.branch == 'develop'
+#        continue-on-error: true
+#        run: |-
+#          ./mvnw -B -Dstyle.color=always \
+#          -Dsonar.projectKey=judo-jsl-springboot-parent \
+#          -Dsonar.host.url=https://sonar.judo.technology \
+#          -Dsonar.login=${{ secrets.JUDO_SONAR_TOKEN }} \
+#          sonar:sonar
```
Comment (robertcsakany) [inline .github/workflows/build.yml]: Why it is removed?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -119,22 +119,22 @@ jobs:
...
-          -Dsonar.projectKey=judo-jsl-springboot-parent \
-          -Dsonar.host.url=https://sonar.judo.technology \
-          -Dsonar.login=${{ secrets.JUDO_SONAR_TOKEN }} \
-          sonar:sonar
+#      - name: 🔎 Sonar Metrics
+#        if: steps.context.outputs.branch == 'develop'
+#        continue-on-error: true
+#        run: |-
+#          ./mvnw -B -Dstyle.color=always \
+#          -Dsonar.projectKey=judo-jsl-springboot-parent \
```
Comment (bencegelei) [inline .github/workflows/build.yml]: > -Dsonar.projectKey=judo-jsl-springboot-parent

Shouldn't this be `judo-jsl-springboot-starter`

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (gaborflorian) [reply .github/workflows/build.yml]: No java files in the project

File: .github/workflows/build.yml (hunk 2 above)
Comment (gaborflorian) [reply .github/workflows/build.yml]: [b330f6c](https://github.com/BlackBeltTechnology/judo-jsl-springboot-starter/pull/21/commits/b330f6c17ea00b2d9a40452cf6061a0e9d8cd0d9)

**Review by robertcsakany — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/2-feature.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ labels: "type: feature"
 <!-- Please do your best to fill out all of the sections below! -->
 <!-- Use this issue type for concrete suggestions, otherwise, open a discussion type issue instead. -->
 
-- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/BlackBeltTechnology/judo-jsl-springboot-starter/blob/develop/CONTRIBUTING.md))
+- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/BlackBeltTechnology/judo-jsl-springboot-starter/blob/develop/CONTRIBUTING.adoc))
```
Comment (fmagnucz) [inline .github/ISSUE_TEMPLATE/2-feature.md:10]: @gaborflorian Is this OK? From a md file link an adoc?

Comment (fmagnucz) [reply .github/ISSUE_TEMPLATE/2-feature.md:10]: We discussed.

**Review by fmagnucz — APPROVED** [review]

**Pull request merged**